### PR TITLE
Type Token documentation 💪

### PR DESCRIPTION
**Pull request by ntmedina**
Add type token documentation to VTEX Styleguide Styles

### CODE REVIEW

File: docs/styles/typography.md [hunk 1]
```diff
@@ -1,47 +1,77 @@
...
-
-```sh
-.f1 { font-size: 3rem; }
-.f2 { font-size: 2.25rem; }
-.f3 { font-size: 1.5rem; }
-.f4 { font-size: 1.25rem; }
-.f5 { font-size: 1rem; }
-.f6 { font-size: 0.875rem; }
-.f7 { font-size: 0.75rem; }
+<div class="pb7"></div>
```
Comment (kevinch) [inline docs/styles/typography.md]: Useful?

File: docs/styles/typography.md [hunk 2]
```diff
@@ -1,47 +1,77 @@
...
+<div class="f2 c-on-base pv5">Typography</div>
+Typography is used to communicate informations the most efficient way possible through legibility and visual hierarchy. It's a crucial tool to guide users on their tasks. It should be used on clear and delightful way. For that, it's important to define different **typography styles** for an app by setting a combination of different typography attributes: **Typeface, weight, size, capitalization and letter spacing.**
+
+Typography styles work as single-purpose CSS classes used to modify the typography configuration of an app. Each class configures the following CSS attributes: ** font-family, font-size, font-weight, text-transform and letter-spacing. **
+
+<!-- Headings -->
+<div class="flex mt7 mb3 items-center">
```
Comment (kevinch) [inline docs/styles/typography.md]: Isn't `tc` (text-center) enough in this case?

File: docs/styles/typography.md [hunk 3]
```diff
@@ -1,47 +1,77 @@
...
+</div>
+Headings are the most prominent type elements of an app. They are the first elements that users read. They should call attention and give information the most efficient way, using no more than the necessary quantity of words. Heading-1 is the most prominent heading style and heading-6 is the less one. Headings should be applied to:
+
+<div class="pa3 br2 bg-muted-5 c-on-muted-5 mb5 dib mr5">Product names</div>
+<div class="pa3 br2 bg-muted-5 c-on-muted-5 mb5 dib mr5">Prices</div>
+<div class="pa3 br2 bg-muted-5 c-on-muted-5 mb5 dib">Page titles</div>
+<div class="pa3 br2 bg-muted-5 c-on-muted-5 mb5 dib">Input labels</div>
+
+```js
+<div className="">
```
Comment (kevinch) [inline docs/styles/typography.md]: Remove empty attribute.

File: docs/styles/typography.md [hunk 4]
```diff
@@ -1,47 +1,77 @@
...
+  <h3 className="t-heading-3">Level 3 Heading</h3>
+  <h4 className="t-heading-4">Level 4 Heading</h4>
+  <h5 className="t-heading-5">Level 5 Heading</h5>
+  <h6 className="t-heading-6">Level 5 Heading</h6>
+</div>
 ```
 
-### Media Query Extensions:
+<!-- Body -->
+<div class="flex mt7 mb3 items-center">
```
Comment (kevinch) [inline docs/styles/typography.md]: Same here.

File: docs/styles/typography.md [hunk 5]
```diff
@@ -1,47 +1,77 @@
...
+    <div class="f3 fw5 c-on-base mv0 mr5 pv3">Body</div>
+</div>
+Body is the text style defined to achieve a good level of rhythm and legibility on longer texts. Paragraph should be applied to:
+
+<div class="pa3 br2 bg-muted-5 c-on-muted-5 mb5 dib mr5">Product descriptions </div>
+<div class="pa3 br2 bg-muted-5 c-on-muted-5 mb5 dib mr5">Forms</div>
+<div class="pa3 br2 bg-muted-5 c-on-muted-5 mb5 dib">Reports</div>
+
+```js
+<div className="">
```
Comment (kevinch) [inline docs/styles/typography.md]: Idem.

File: docs/styles/typography.md (hunk 1 above)
Comment (ntmedina) [reply docs/styles/typography.md]: Fixed.

File: docs/styles/typography.md (hunk 2 above)
Comment (ntmedina) [reply docs/styles/typography.md]: Fixed.

File: docs/styles/typography.md (hunk 3 above)
Comment (ntmedina) [reply docs/styles/typography.md]: Fixed.

File: docs/styles/typography.md (hunk 4 above)
Comment (ntmedina) [reply docs/styles/typography.md]: Fixed.

File: docs/styles/typography.md (hunk 5 above)
Comment (ntmedina) [reply docs/styles/typography.md]: Fixed. Thaanx 🙌🏼

**Comment by klzns** [comment]
Duplicate of #441

**Pull request closed without merging**